### ISSUE TITLE
DataDog support

### ISSUE THREAD

**Issue by RaviVaranasi** [jongpie/NebulaLogger#466]
### New Feature Summary

Can I ship logs to DataDog instead of logging inside Salesforce? The advantage is to have one logging system across multiple systems

**Comment by jamessimone** [comment]
@RaviVaranasi I would look at the [Slack plugin](https://github.com/jongpie/NebulaLogger/tree/main/nebula-logger/plugins/slack) for the canonical way to publish log entries off-platform.

- I'm sure @jongpie can add additional info - and perhaps an official DataDog plugin can also be supported - but if you're looking to build out this functionality and the question is just "is it possible," the architecture in the Slack plugin should be enough to get going!

**Comment by jamessimone** [comment]
You still can!

**Comment by sjurgis** [comment]
@jongpie any chance you know any resources / examples how to log apex metrics (mostly around performance)?

**Comment by sjurgis** [comment]
FWIW Datadog does have a connector for Event Monitoring https://docs.datadoghq.com/integrations/salesforce/

**Comment by RaviVaranasi** [comment]
> FWIW Datadog does have a connector for Event Monitoring https://docs.datadoghq.com/integrations/salesforce/

Does this do logging?

**Comment by jonathankretzmer** [comment]
Instead of actively targeting a single provider like DataDog, it would possibly be more beneficial to target an open standard, e.g. OpenTelemetry (which DataDog supports).

**Comment by jongpie** [comment]
@jonathankretzmer I agree, I think it makes more sense to target the OpenTelemetry standard, instead of vendor-specific integrations. There was [recently a discussion](https://github.com/jongpie/NebulaLogger/discussions/479) about this too, so I think a lot of people would benefit from using OpenTelemetry. I've started some initial analysis & prototyping on this already - I'm hoping to work on it more over the next several weeks.
